modules/installation-ibm-z-troubleshooting-and-debugging.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-ibm-z-troubleshooting-and-debugging_{context}"]
+= Completing installation on user-provisioned infrastructure
```

Comment:
I'm really confused as to what this module is supposed to be. If you look at the assembly for installing on IBM Z/LinuxOne, it shows the same thing twice:

```
Completing installation on user-provisioned infrastructure
Completing installation on user-provisioned infrastructure
```

But it does look like it's supposed to be troubleshooting/debugging info like the file is named. Is the title just wrong?

Reply:
That's right, I somehow used the wrong title. Fixed to be "Collecting debugging information".